pkg/plugins/runtime/k8s/controllers/pod_converter_test.go

```diff
@@ -844,7 +844,7 @@ var _ = Describe("ProtocolTagFor(..)", func() {
 		}),
 		Entry("appProtocol has an unknown value", testCase{
 			appProtocol: utilpointer.String("not-yet-supported-protocol"),
-			expected:    "not-yet-supported-protocol", // we want Kuma's behavior to be straightforward to a user (just copy appProtocol lowercase value)
+			expected:    "tcp", // we want Kuma's behavior to be straightforward to a user (appProtocol is not Kuma specific)
```

Comment:
so isn't this a breaking change?

Reply:
Well these were all invalid `Dataplane`s before

Reply:
and now they're valid... I guess this is ok.